java/pom.xml

```diff
@@ -131,7 +132,7 @@
           <executions>
             <execution>
               <id>attach-sources</id>
-              <phase>deploy</phase>
+              <phase>package</phase>
```

Comment:
what's the implication of this change?

Reply:
actually, this is the best practice of maven source plugin. 

we need to read the ray code in development, so when run `mvn install`, it will also install ray-xxx-source.jar into local maven repo. It helps a lot for debugging.

Reply:
thanks for explaining. that's indeed what we want.